docs/docs/guides/spaces.mdx

```diff
@@ -78,53 +78,55 @@ Deleting a Space will delete all of the content in that Space.
 
 ## Managing access to a Space
 
-By default, when you create a Space, it is restricted so only you can access it. You can edit a Space to make it Restricted Access or Full Access.
+By default, when you create a Space, it is restricted so only you can access it. You can edit a Space to make it Restricted Access or Public.
 
-To edit access to a Space, navigate into the Space, then click the `share` button. From there, you can choose to make a space `Restricted Access` or `Full Access`.
+To edit access to a Space, navigate into the Space, then click the `share` button. From there, you can choose to make a space `Restricted Access` or `Public`.
 
 <img
   src={EditSpaceAccess}
-  width="2249"
-  height="445"
+  width="1611"
+  height="397"
   style={{ display: 'block', margin: '0 auto 20px auto' }}
 />
 
-Users inherit their space permissions from their project permissions. For more details about actions you can take in a Space based on your role, [check out this table](/references/roles.mdx#space-roles).
+By default, in public spaces, a user's space permissions are inherited from their project permissions. For example, if I'm a project viewer, I will get `Can View` access to a Space.
+
+There are three space roles: `Full Access`, `Can Edit`, `Can View`
+
+For more details about actions you can take in a Space based on your role, [check out this table](/references/roles.mdx#space-roles).
 
 ### Restricted Access Spaces
 
 **Only admins and members that are invited to a Space with Restricted Access have access to it.**
 
-For example, here I have a Space called `Finance` with Restricted Access. Only admins and the invited members (Katie, who created the space, and Hamzah, who was invited) have access to this Space. No one else in Lightdash can see or interact with the content in this Space unless they're invited or become an admin.
+For example, here I have a Space with Restricted Access. Only admins and the invited members have access to this Space. Katie, Filipe and Marshall have inherited `Full Access` because they are organization admins. Priyanka was invited to the space, so she has `Can View` access to the space. No one else in Lightdash can see or interact with the content in this Space unless they're invited or become an admin.
+
+Only users with `Full Access` permissions to a Restricted Space can invite other users to that Space.
 
 <img
   src={RestrictedAccessSpace}
-  width="540"
-  height="356"
+  width="1292"
+  height="495"
   style={{ display: 'block', margin: '0 auto 20px auto' }}
 />
```

Comment:
The image shows we picked a user with full access and then shows as a Can view. A user inherited Full access cannot be lowered.
Basically, org/project admins cannot have their role restricted in spaces.